modules/games/memoryMatch.ts

```diff
@@ -113,7 +113,16 @@ export default async function memoryMatch(
 							: [options.user, interaction.user],
 					easyMode,
 					bonusTurns,
-					useThread: options.thread ?? true,
+					useThread:
+						options.thread ??
+						(!(
+							await config.guild.members.fetch(options.user.id)
+						).presence?.clientStatus?.mobile ||
```

Comment:
please move the checks to their own variables before the await playGame
i believe you can use `interaction.guild.presences.resolve(options.user.id)`
and then check if `presence.status !== presence?.clientStatus?.mobile` for both users

and then in games/index.ts, please just remove the part about `thread`'s default
thanks!